target_chains/ton/contracts/wrappers/WormholeTest.ts

```diff
@@ -0,0 +1,170 @@
+import {
+  Address,
+  beginCell,
+  Cell,
+  Contract,
+  contractAddress,
+  ContractProvider,
+  Dictionary,
+  Sender,
+  SendMode,
+} from "@ton/core";
+import { createCellChain } from "../tests/utils";
+
+export type WormholeTestConfig = {
+  guardianSetIndex: number;
+  guardianSet: string[];
+  chainId: number;
+  governanceChainId: number;
+  governanceContract: string;
+};
+
+export function pythConfigToCell(config: WormholeTestConfig): Cell {
```

Comment:
I think we can remove this function and createFromConfig can just call getWormholeInitData